CHANGELOG

```diff
@@ -39,6 +40,22 @@ pybats
  - Updates to ram.RamSat including omnidirectional flux calculation
  - ram.PressureFile fixed for Python 3.x failures
  - rim.Iono objects now load poorly-formatted files; rim.fix_format deprecated
+time
+ - Fix handling of leapseconds before 1961.
+ - For types that cannot represent leapseconds (e.g. UTC datetime), time
+   within the leap second is now converted to the end of the day rather
+   than the beginning of the next day.
+ - Fractional leapsecond counts are now rounded to the integer rather than
+   truncated.
+ - Fix conversions to ISO before 1900 on Python 2.
+ - Include leapseconds in JD/MJD calculation, so the length of a second on a
+   day with leapseconds is slightly shorter but leap second times are not lost.
+ - Most time systems are now converted to/from TAI. This may cause small
+   differences with previous versions of SpacePy, on order of a double
+   precision. Time conversions around leapseconds may also be different;
```

Comment:
On order of a double precision isn't going to mean much to a lot of users. This should be less than a millisecond, right?